maltiverse/info.json

```diff
@@ -0,0 +1,264 @@
+{
+    "name": "maltiverse",
+    "label": "Maltiverse",
+    "version": "1.0.0",
+    "description": "Maltiverse Threat Intelligence Feeds can be integrated with your security stack to provide improvement in terms of detections and protection capabilities from different points of view. You can also upload an deploy your own Threat Intelligence!",
+    "publisher": "Fortinet",
+    "cs_approved": false,
```

Comment:
I think this connector will be certified, so you can mark this as true

Reply:
marked it as true